lib/hexpm/factory.ex

```diff
@@ -55,6 +55,15 @@ defmodule Hexpm.Factory do
     }
   end
 
+  def audit_log_factory() do
+    %Hexpm.Accounts.AuditLog{
+      action: "NOTE: this is the default value set in Hexpm.Factory.audit_log_factory/0",
+      params: %{
+        notice: "NOTE: this is the default value set in Hexpm.Factory.audit_log_factory/0"
```

Comment:
Do we need this? I would rather have a proper default or leave them blank.

Reply:
I'm okay with a proper default. I'll make them blank.

(It's actually the first time I use this note to make the default value more standout. 😂 )

Reply:
updated in https://github.com/hexpm/hexpm/pull/817/commits/fce84d107ac3e77f86b5052ee7d22c1f0fb3d57c